frontends/infinite-corridor/src/pages/ForbiddenPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import Container from "@mui/material/Container"
+import Card from "@mui/material/Card"
+import CardContent from "@mui/material/CardContent"
+import CardActions from "@mui/material/CardActions"
+import Button from "@mui/material/Button"
+
+const ForbiddenPage: React.FC = () => {
+  return (
+    <Container maxWidth="sm">
+      <Card style={{ marginTop: "1em" }}>
```

Comment:
It's not really relevant here, but fyi there is also `sx`: https://mui.com/system/getting-started/the-sx-prop/

`sx` is mui specific, whereas `style` is... just `style`. The main differences are (1) sx uses classes internally to apply the css, which means it can be overriden and it can use things like pseudoselectors; (2) `sx` has access to MUI's theme.